config/checkstyle/checkstyle.xml

```diff
@@ -1,248 +1,237 @@
 <?xml version="1.0"?>
```

Comment:
Taken from HMCTS rules with two changes. TBC whether a better way to "extend" the original instead of having the whole thing here.